astropy/coordinates/solar_system.py

```diff
@@ -149,25 +151,29 @@ def _get_kernel(value):
     if value is None or value.lower() == 'builtin':
         return None
 
+    try:
+        from jplephem.spk import SPK
+    except ImportError:
+        raise ImportError("Solar system JPL ephemeris calculations require "
+                          "the jplephem package "
+                          "(https://pypi.python.org/pypi/jplephem)")
+
     if value.lower() == 'jpl':
         value = DEFAULT_JPL_EPHEMERIS
 
-    if value.lower() in ('de430', 'de432s'):
+    elif value.lower() in ('de430', 'de432s'):
```

Comment:
This should remain an `if` - since `DEFAULT_JPL_EPHEMERIS = 'de430'`

I guess I never added a test since I didn't want to download an enormous file for every test run...